lib/platform.mli

```diff
@@ -13,16 +13,19 @@ val pp : t Fmt.t
 val compare : t -> t -> int
 
 val get :
+  arch:Ocaml_version.arch option ->
   label:string ->
   builder:Builder.t ->
   pool:string ->
   distro:string ->
   ocaml_version:string ->
+  Current_docker.Raw.Image.t Current.t *
```

Comment:
Would be clearer to use a labelled argument here I think. I try to avoid having two unnamed arguments with same type and no obvious order.

Reply:
done in c871088a8137d1f2c5d764172669497e53e14f59